content-services/latest/admin/control-center.md

```diff
@@ -0,0 +1,211 @@
+
+    If you already have a valid license file for Content Services 7.2, you can apply it directly to the running system. See [Uploading a new license]({% link content-services/latest/admin/license.md %}) for more details.
+
+    > **Note:** Make sure that exposed ports are open on your host computer. Check the `docker-compose.yml` file to determine the exposed ports - refer to the `host:container` port definitions. You'll see they include 5432, 8080, 8083 and others.
+
+    > **Note:** The Download Trial is usually updated for *major.minor* versions of Content Services. The latest published version on our website is labelled *Version 7.2 - March 2022)*.
+
+2. Save the `docker-compose.yml` file in a local folder.
+
+3. Log in to Quay.io using your credentials:
+
+    ```bash
+    docker login quay.io
+    ```
+
+    Alfresco customers can request Quay.io credentials by logging a support ticket via [Hyland Community](https://community.hyland.com/){:target="_blank"}. These credentials are required to pull private (Enterprise-only) Docker images from Quay.io.
+
+4. Deploy Content Services, including the repository, Share, Postgres database, Search Services, etc.:
+
+    ```bash
+    docker-compose up
+    ```
+
+    This downloads the images, fetches all the dependencies, creates each container, and then starts the system.
+
+    As an alternative, you can also start the containers in the background by running `docker-compose up -d`.
+
+5. Wait for the logs to complete.
+
+    See [Troubleshooting](#troubleshooting) if you encounter errors whilst the system is starting up.
+
+6. Open your browser and check everything starts up correctly:
+
+    | Service | Endpoint |
+    | ------- | -------- |
+    | Control Center | `http://localhost:8080/admin` |
+    | Digital Workspace | `http://localhost:8080/workspace` |
+    | Share | `http://localhost:8080/share` |
+    | Administration and REST APIs | `http://localhost:8080/alfresco` |
+
+    If Docker is running on your local machine, the IP address will be just `localhost`.
+
+    If you're using the [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_windows/){:target="_blank"}, run the following command to find the IP address:
+
+    ```bash
+    docker-machine ip
+    ```
+
+7. Log in to the Control Center as the `admin` user. Enter the default administrator password `admin`.
+
+## Manage users
+
+The **Identity** section of the application is used to create and manage users, and groups. Expand the section to see the two main areas the Control Center covers: **Users** and **Groups**.
+
+The **Users** section displays the current list of users in the system.
+
+### Properties {#user-properties}
+
+The properties for users are:
+
+| Property | Description |
+| -------- | ----------- |
+| ID | A unique identifier for the user. This is system generated and cannot be changed. |
+| Username | A username for the user. |
```

Comment:
This does not look correct for current version of the Alfresco Control Center 7.2.0 (ie. when running with ACS Repo users & groups). Maybe change to:

| Id / Username | A unique identifier (username) for the user. |

Reply:
OK, I see the confusion now! 🙄 I looked at what I thought was a system-generated user first. Now I've tested it and I can see the difference. Text revised.